install/source/chapter10.rst

```diff
@@ -12,7 +12,7 @@ the web portal as the Master Administrator.
 
 After you login to the web portal, click to the "Cluster Branding" Pages.
 
-.. image:: _static/image_s10_1_1.png
+.. image:: _static/image_s10_1_1_v2.png
 
 Please reference the "Branding" section in the `Admin Guide`__ for the self-hosted CentreStack server.
```

Comment:
Point this Admin Guide link to the specific section for the branding in the Admin Guide:
https://centrestack.com/Library/AdminGuide/chapter03.html#cluster-branding